Drop pouches for medical + name refactor

x5tink opened the pull request:
## About the PR
While looking at something else i noticed that in CM13 medical staff have access to the drop pouch in their vendors, so this adds that, Also i renamed the Drop pouch from CMWebbingPouch to CMWebbingDropPouch for clarity, as i got awful confused when i looked at it, and this is a bit clearer.

## Why / Balance
Parity and Clarity... Hey that ryhmes....

## Technical details
Renamed all the CMWebbingPouch to CMWebbingDropPouch for clarity
Added drop pouch to medical vendors

## Media
Before:
![DropPouchBefore](https://github.com/user-attachments/assets/4eb63f9c-c4e2-4ae0-a56a-c32241a3995e)

After:
![DropPouchAfter](https://github.com/user-attachments/assets/8f7b1883-af3c-4b87-b9ab-401368dd8d7d)

PR Rename:
![image](https://github.com/user-attachments/assets/236c6b98-c365-4aa9-a327-c5a94e4bd207)


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->
- [x] By submitting this code and/or assets, I confirm that I either own them or have provided the correct necessary licenses to use and distribute them. I agree to be fully responsible for any legal claims or issues arising from the use of these materials.

**Changelog**
:cl:
- add: Added drop pouch to medical vendors

DrSmugleaf reviewed (changes requested):
Thanks for the PR!
You don't need to edit existing map files, you can just add CMWebbingPouch: CMWebbingDropPouch in migration.yml to automatically change it in any existing maps next time they are saved, and when they are loaded

x5tink commented:
I was not aware the migrations thing was an uh... thing xD I've removed the map renames, added to the migration file as you suggested.
I Then renamed them all to RMCWebbingDropPouch, to be more inline with the other renames, but noticed some inconsistantcy on the names, in the migrations file. Is there a specific reason to name something CM vs RMC?

x5tink commented:
Should have fixed the conflict....

DrSmugleaf commented:
> I was not aware the migrations thing was an uh... thing xD I've removed the map renames, added to the migration file as you suggested. I Then renamed them all to RMCWebbingDropPouch, to be more inline with the other renames, but noticed some inconsistency in the names, in the migrations file. Is there a specific reason to name something CM vs RMC?

CM is old IDs, any new IDs that aren't for job prototypes specifically should be prefixed RMC

x5tink commented:
Rebased and fixed the migrations file again.
